2024-09-20 04:15:24,537 INFO    MainThread:1951526 [wandb_setup.py:_flush():77] Current SDK version is 0.17.8
2024-09-20 04:15:24,537 INFO    MainThread:1951526 [wandb_setup.py:_flush():77] Configure stats pid to 1951526
2024-09-20 04:15:24,537 INFO    MainThread:1951526 [wandb_setup.py:_flush():77] Loading settings from /home/geo_llm/.config/wandb/settings
2024-09-20 04:15:24,537 INFO    MainThread:1951526 [wandb_setup.py:_flush():77] Loading settings from /home/geo_llm/llama3-finetune/wandb/settings
2024-09-20 04:15:24,537 INFO    MainThread:1951526 [wandb_setup.py:_flush():77] Loading settings from environment variables: {}
2024-09-20 04:15:24,537 INFO    MainThread:1951526 [wandb_setup.py:_flush():77] Applying setup settings: {'_disable_service': False}
2024-09-20 04:15:24,537 INFO    MainThread:1951526 [wandb_setup.py:_flush():77] Inferring run settings from compute environment: {'program_relpath': 'main.py', 'program_abspath': '/home/geo_llm/llama3-finetune/main.py', 'program': '/home/geo_llm/llama3-finetune/main.py'}
2024-09-20 04:15:24,538 INFO    MainThread:1951526 [wandb_setup.py:_flush():77] Applying login settings: {}
2024-09-20 04:15:24,538 INFO    MainThread:1951526 [wandb_init.py:_log_setup():524] Logging user logs to /home/geo_llm/llama3-finetune/wandb/run-20240920_041524-3yl8u3bo/logs/debug.log
2024-09-20 04:15:24,538 INFO    MainThread:1951526 [wandb_init.py:_log_setup():525] Logging internal logs to /home/geo_llm/llama3-finetune/wandb/run-20240920_041524-3yl8u3bo/logs/debug-internal.log
2024-09-20 04:15:24,538 INFO    MainThread:1951526 [wandb_init.py:init():607] calling init triggers
2024-09-20 04:15:24,538 INFO    MainThread:1951526 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {}
2024-09-20 04:15:24,538 INFO    MainThread:1951526 [wandb_init.py:init():657] starting backend
2024-09-20 04:15:24,538 INFO    MainThread:1951526 [wandb_init.py:init():661] setting up manager
2024-09-20 04:15:24,540 INFO    MainThread:1951526 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-09-20 04:15:24,541 INFO    MainThread:1951526 [wandb_init.py:init():669] backend started and connected
2024-09-20 04:15:24,542 INFO    MainThread:1951526 [wandb_init.py:init():767] updated telemetry
2024-09-20 04:15:24,548 INFO    MainThread:1951526 [wandb_init.py:init():800] communicating run to backend with 90.0 second timeout
2024-09-20 04:15:25,074 INFO    MainThread:1951526 [wandb_init.py:init():851] starting run threads in backend
2024-09-20 04:15:29,648 INFO    MainThread:1951526 [wandb_run.py:_console_start():2463] atexit reg
2024-09-20 04:15:29,649 INFO    MainThread:1951526 [wandb_run.py:_redirect():2309] redirect: wrap_raw
2024-09-20 04:15:29,649 INFO    MainThread:1951526 [wandb_run.py:_redirect():2374] Wrapping output streams.
2024-09-20 04:15:29,649 INFO    MainThread:1951526 [wandb_run.py:_redirect():2399] Redirects installed.
2024-09-20 04:15:29,651 INFO    MainThread:1951526 [wandb_init.py:init():894] run started, returning control to user process
2024-09-20 04:15:56,187 WARNING MsgRouterThr:1951526 [router.py:message_loop():77] message_loop has been closed
